flake.nix

```
@@ -0,0 +1,27 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs = {nixpkgs, ...}: let
+    system = "x86_64-linux";
+    pkgs = nixpkgs.legacyPackages.${system};
+  in {
+    devShells.${system}.default = pkgs.mkShell {
```

suggestion (non-blocking): I have switched to

```suggestion
    devShells.${system}.default = pkgs.mkShellNoCC {
```

in my flake but I'm not sure if it has any advantages (https://nixos.org/manual/nixpkgs/stable/#sec-pkgs-mkShell-variants).

---

Me neither.